src/pages/OutebreakMap/index.js

```diff
@@ -0,0 +1,16 @@
+import { MAP_CONFIG } from "../../data/constants";
+
+export const drawMap = (mapboxgl, map) => {
```

Comment:
This could be better if it was merged with drawMapPrefectures.js and just contained all the things that makes the map work. Splitting the two parts to initialize the map and draw the prefectures seem like it's a little too much separation.